### DISCUSSION TITLE
Next.js starter with Zustand v4

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [pmndrs/zustand#1594] [Q&A]
Hey ! Does someone know a great starter in order to use zustand (v4) and nextjs ? (examples from nextjs and docs are not up to date)

**Comment by rodolphoasb** [comment]
Hey, @MatteoGauthier! Not necessarily a starter but this blog post from @TkDodo is awesome (there are great discussions in the comment section too)!  https://tkdodo.eu/blog/working-with-zustand

Hope it helps!
